client/web/src/enterprise/campaigns/list/CampaignsListEmpty.tsx

```diff
@@ -0,0 +1,214 @@
+                            </h4>
+                            <p className="text-muted mb-0">
+                                Using comby language aware search to refactor Go statements to a semantically
+                                equivalent, but clearer execution.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <p>
+            <a href="https://docs.sourcegraph.com/campaigns/tutorials" rel="noopener">
+                More tutorials
+            </a>
+        </p>
+        <div className="row mb-4">
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Quickstart</strong>
+                </p>
+                <p>Create your first Sourcegraph campaign in 10 minutes or less by altering a readme.</p>
+                <a href="https://docs.sourcegraph.com/campaigns/quickstart" rel="noopener">
+                    Campaigns quickstart
+                </a>
+            </div>
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Introduction</strong>
+                </p>
+                <p>Learn how campaigns enables large-scale code changes across many repositories and code hosts.</p>
+                <a href="https://docs.sourcegraph.com/campaigns/explanations/introduction_to_campaigns" rel="noopener">
+                    Campaigns introduction
+                </a>
+            </div>
+            <div className="col-12 col-md-4 mb-3">
+                <p>
+                    <strong>Documentation</strong>
+                </p>
+                <p>
+                    Learn about the campaign spec{' '}
+                    <a
+                        href="https://docs.sourcegraph.com/campaigns/references/campaign_spec_yaml_reference"
+                        rel="noopener"
+                    >
+                        YAML reference
+                    </a>
+                    , it's powerful{' '}
+                    <a href="https://docs.sourcegraph.com/campaigns/references/campaign_spec_templating">
+                        templating language
+                    </a>
+                    ,{' '}
+                    <a
+                        href="https://docs.sourcegraph.com/campaigns/explanations/permissions_in_campaigns"
+                        rel="noopener"
+                    >
+                        permissions
+                    </a>{' '}
+                    and more in the{' '}
+                    <a href="https://docs.sourcegraph.com/campaigns" rel="noopener">
+                        campaigns documentation
+                    </a>
+                    .
+                </p>
+            </div>
+        </div>
+        <h2>Campaigns demo</h2>
+        <p className="text-muted">
+            This demo highlights the ease of refactoring code and managing changesets across many repositories and
```

Comment:
"highlights the ease of refactoring code" is not true, I'd say. It also reads a bit marketing-y, so I suggest making it plain:

```suggestion
            This demo shows how to refactor code and manage changesets across many repositories and
```